aws/logs_monitoring/lambda_function.py

```diff
@@ -122,11 +122,34 @@ def __init__(self, name, pattern, placeholder):
             "(?<!^)\s+(?={})(?!.\s)".format(DD_MULTILINE_LOG_REGEX_PATTERN)
         )
     except Exception:
-        raise Exception("could not compile multiline regex with pattern: {}".format(DD_MULTILINE_LOG_REGEX_PATTERN))
+        raise Exception("could not compile s3 multiline regex with pattern: {}".format(DD_MULTILINE_LOG_REGEX_PATTERN))
     multiline_regex_start_pattern = re.compile(
         "^{}".format(DD_MULTILINE_LOG_REGEX_PATTERN)
     )
 
+
+# DD_MULTILINE_CLOUDWATCH_REGEX: Datadog Multiline Cloudwatch Logs Regular Expression Patterns for Log Groups
+DD_MULTILINE_CLOUDWATCH_LOG_REGEX_PATTERNS = None
```

Comment:
Do you think we might expect other AWS products to require multiline regexes ? If so, do we want to create a new environment variable for each product ?

Reply:
I believe the other options here that we differentiate on would be `aws sns` or `aws kenesis`  logs, while i'm not familiar with them I assume they could also contain multiline logs. My thinking was that if we implement handling multiline logging for either of those we would add their own environment variable at that time.